lib/core/presentation/theming/themes/light_theme.dart

```diff
@@ -63,7 +73,7 @@ ThemeData whiteTheme(BuildContext context, BoxConstraints cts) {
       ),
       titleSmall: style.titleSmall?.copyWith(
         fontSize: cts.maxWidth * 0.04,
-        color: _AppColors.onSurfaceW,
+        color: _AppColors.greyLight,
```

Comment:
- The greyLight color code doesn't match the two flavors on the design `#CACACA` and `#E0E0E0`
- `lightGrey` would make a better name